google/cloud/spanner/internal/connection_impl_test.cc

```diff
@@ -2524,6 +2533,31 @@ TEST(ConnectionImplTest, CommitSuccessWithStats) {
   EXPECT_EQ(42, commit->commit_stats->mutation_count);
 }
 
+TEST(ConnectionImplTest, CommitSuccessWithMaxCommitDelay) {
+  auto mock = std::make_shared<spanner_testing::MockSpannerStub>();
+  auto db = spanner::Database("placeholder_project", "placeholder_instance",
+                              "placeholder_database_id");
+  EXPECT_CALL(*mock, BatchCreateSessions(_, HasDatabase(db)))
+      .WillOnce(Return(MakeSessionsResponse({"test-session-name"})));
+  google::spanner::v1::Transaction txn = MakeTestTransaction();
+  EXPECT_CALL(*mock, BeginTransaction).WillOnce(Return(txn));
+  EXPECT_CALL(
+      *mock,
+      Commit(_, AllOf(HasSession("test-session-name"),
```

Comment:
`s/_,/_, _,/` and a re-clangformat

You'll need an extra argument here to allow for a change to the `Commit()` API since this was added.

Reply:
Done.